app/pages/sections/catalogue-solutions/order-item/manifest.json

```diff
@@ -37,6 +37,35 @@
       }
     }
   },
+  "columnInfo": [
+    {
+      "data": "Price (£)"
+    },
+    {
+      "data": "Unit of order"
+    }
+  ],
+  "data": [
```

Comment:
take it this will be removed when you come to do the populateData task?

Reply:
yeah was just temp data